cms/src/components/forms/LCombobox.vue

```diff
@@ -0,0 +1,143 @@
+<script setup lang="ts">
+import { computed, ref, watch, type StyleValue } from "vue";
+import { ChevronUpDownIcon } from "@heroicons/vue/20/solid";
+import LTag from "../content/LTag.vue";
+import { useAttrsWithoutStyles } from "@/composables/attrsWithoutStyles";
+import FormLabel from "./FormLabel.vue";
+import LInput from "./LInput.vue";
+import { onClickOutside } from "@vueuse/core";
+
+export type ComboboxOption = {
+    id: string | number;
+    label: string;
+    value: any;
+};
+
+type Props = {
+    label?: string;
+    disabled?: boolean;
+    options: ComboboxOption[];
+};
+
+const props = withDefaults(defineProps<Props>(), {
+    disabled: false,
+});
+
+//const selected = ref<Array<string | number>>([])
+
+const selectedOptions = defineModel<Array<string | number>>("selectedOptions");
+
+const optionsList = computed(() =>
+    props.options.map((o) => ({
+        id: o.id,
+        label: o.label,
+        value: o.value,
+        selected: selectedOptions.value?.includes(o.id),
+    })),
+);
+const optionsToDisplay = ref(optionsList.value);
+
+const query = ref("");
+
+watch(query, () => {
+    const currentOptionsList = optionsList.value;
+    if (query.value.length > 0) {
+        showOptions.value = true;
+
+        // Filter options based on query matching the label (case-insensitive)
+        optionsToDisplay.value = currentOptionsList.filter((o) =>
+            o.label.toLowerCase().includes(query.value.toLowerCase()),
+        );
+    } else {
+        optionsToDisplay.value = currentOptionsList;
+    }
+});
+
+const input = ref();
+
+const optionsDisplay = ref();
```

Comment:
The naming is a bit confusing (as we also have an `optionsToDisplay`). Perhaps something like `optionsElement` or `optionsDiv` could work?